NuRadioReco/modules/io/eventWriter.py

```diff
@@ -83,7 +83,7 @@ def run(self, evt, det = None, mode='full'):
         if(mode not in ['full', 'mini', 'micro']):
             logger.error("output mode must be one of ['full', 'mini', 'micro'] but is {}".format(mode))
             raise NotImplementedError
-        self.__check_for_duplicate_ids(evt.get_run_number(), evt.get_id())
+#         self.__check_for_duplicate_ids(evt.get_run_number(), evt.get_id())
```

Comment:
Why did you comment this out?

Reply:
ups, this was unintentional,thanks for spotting